packages/blade/src/components/ChatMessage/docs/ChatMessage.stories.tsx

```diff
@@ -0,0 +1,270 @@
+import type { StoryFn, Meta } from '@storybook/react';
+import React from 'react';
+import { m } from 'framer-motion';
+import { ChatMessage } from '../ChatMessage';
+import type { ChatMessageProps } from '../types';
+import { Heading } from '~components/Typography/Heading';
+import { Box } from '~components/Box';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { RayIcon } from '~components/Icons';
+import { Card, CardBody } from '~components/Card';
+import { Text } from '~components/Typography';
+import { Radio, RadioGroup } from '~components/Radio';
+import { Move } from '~components/Move';
+import { Chip, ChipGroup } from '~components/Chip';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="ChatMessage"
+      componentDescription="A Chat Message is a visual representation of a message in a chat application."
+      apiDecisionLink={null}
+      figmaURL="https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=100413-32686&t=n9A7LztwEkIsly3v-0"
+    >
+      <Heading size="large">Usage</Heading>
+      <Sandbox showConsole>
+        {`
+        import { ChatMessage } from '@razorpay/blade/components';
+        
+        function App() {
+          return (
+            <ChatMessage > Hi, from ray! </ChatMessage>
+          )
+        }
+
+        export default App;
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/ChatMessage',
+  component: ChatMessage,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<ChatMessageProps>;
+
+const ChatMessageTemplate: StoryFn<typeof ChatMessage> = () => {
+  return (
+    <ChatMessage senderType="self" messageType="last">
```

Comment:
```suggestion
const ChatMessageTemplate: StoryFn<typeof ChatMessage> = (args) => {
  return (
    <ChatMessage {...args}>
```

This plus
```jsx
export const Default = ChatMessageTemplate.bind({});
Default.args = {
  senderType: 'self',
  messageType: 'last'
}
```

This will make the storybook arguments work
<img width="781" alt="image" src="https://github.com/user-attachments/assets/6c4d2c08-58f3-4ae1-9f7b-341d6bc8a498" />